.storybook/main.ts

```diff
@@ -2,12 +2,25 @@ import type { StorybookConfig } from '@storybook/react-vite';
 
 const config: StorybookConfig = {
   stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
+  typescript: {
+    reactDocgen: 'react-docgen-typescript',
+    reactDocgenTypescriptOptions: {
+      shouldExtractLiteralValuesFromEnum: true,
```

Comment:
```suggestion
      shouldExtractLiteralValuesFromEnum: true,
      shouldRemoveUndefinedFromOptional: true,
```

optional field의 경우 `| undefined`가 붙어있어서 제어할 때 번거로운 거 같아요.
특히 boolean 타입의 경우가 그렇다고 느꼈습니다!!

[react-docgen-typescript의 이슈 코멘트](https://github.com/styleguidist/react-docgen-typescript/issues/81#issuecomment-398403525) 보고 스크립트 작성해야 한다는 말이 이건가? 했는데
찾아보니 해당 이슈 이후로 optional field에 한해서 undefined를 없애주는 옵션이 만들어졌네요.
설정해두는 게 좋을 거 같아 코드 제안 남겨뒀습니다!!

Reply:
| 설정 전 | 설정 후 |
|:---:|:---:|
|![image](https://github.com/yourssu/YDS-React/assets/87255462/7a03194e-6cb8-4695-ae18-8266a9c7c25c)|![image](https://github.com/yourssu/YDS-React/assets/87255462/7be0c196-7a57-40e2-a83b-380dd6255854)|